src/components/Card.js

```diff
@@ -1,9 +1,27 @@
 export default class Card {
-  constructor(data, cardSelector, handleCardClick) {
+  constructor(
+    data,
+    cardSelector,
+    id,
+    handleCardClick,
+    handleCardDelete,
+    handleLike
+  ) {
     this._name = data.name;
     this._link = data.link;
     this._cardSelector = cardSelector;
     this._handleCardClick = handleCardClick;
+    this._owned = data.owner._id === id;
+    this._id = data._id;
+    this._handleCardDelete = handleCardDelete;
+    this._like = data.likes.length;
```

Comment:
There is no need to save this, since the value can be calculated. save the likes array instead `this._likes = likes`